lib/functionTemplates/template.js

```diff
@@ -0,0 +1,28 @@
+const format = require('string-format');
+
+const node8Template =
+`'use strict'
+
+/**
+ * @type {{ import("./binaris").Handler }}
+ */
+exports.{entrypoint} = async (body, context) => {
+  const name = context.request.query.name || body.name || 'World';
+  return \`Hello \${{name}}!\`;
```

Comment:
[LTS](https://en.wikipedia.org/wiki/Leaning_toothpick_syndrome), and I don't mean "Long Term Support".  This line has 5 escape sequences using 3 different escape characters.

Reply:
This is good enough IMO, not ideal but good enough.